src/Corrade/Containers/PairStl.h

```diff
@@ -41,7 +41,22 @@ construction and assignment. See @ref Containers-Pair-stl for more information.
 
 /* Listing these namespaces doesn't add anything to the docs, so don't */
 #ifndef DOXYGEN_GENERATING_OUTPUT
-namespace Corrade { namespace Containers { namespace Implementation {
+
+namespace std {
+template<class F, class S> struct tuple_size<Corrade::Containers::Pair<F, S>> : std::integral_constant<std::size_t, 2> {};
+template<class F, class S> struct tuple_element<0, Corrade::Containers::Pair<F, S>> { using type = F; };
+template<class F, class S> struct tuple_element<1, Corrade::Containers::Pair<F, S>> { using type = S; };
+}
+
+namespace Corrade { namespace Containers {
+
+#if CORRADE_CXX_STANDARD >= 201703
+template<std::size_t N, class F, class S> constexpr const std::tuple_element_t<N, Pair<F, S>> & get(const Pair<F, S>& value) { if constexpr(N == 1) return value.second(); return value.first(); }
+template<std::size_t N, class F, class S> constexpr std::tuple_element_t<N, Pair<F, S>>& get(Pair<F, S>& value) { if constexpr(N == 1) return value.second(); return value.first(); }
+template<std::size_t N, class F, class S> constexpr std::tuple_element_t<N, Pair<F, S>> get(Pair<F, S>&& value) { if constexpr(N == 1) return value.second(); return value.first(); }
```

Comment:
Hmm, now I checked, and there isn't a way around polluting my namespaces with `get()`, right? Sigh.

Somehow I thought `std::tuple_element` specialization would *itself* contain the `get()`, because that would make total sense, and would be quite clean, but apparently not?

That's also why I thought this could be enabled for C++11 too, to make `std::get<N>()` working on these types... but if not, then it just doesn't make sense to have this there. And I don't see any other use cases to have this on C++11, so having it C++17-only is fine.

I need to think a bit whether go with this namespace-polluting way, or whether just go with `get` as a member. I'll get back to you on that.